addons/binding/org.openhab.binding.enocean/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,46 @@
+Manifest-Version: 1.0
+Automatic-Module-Name: org.openhab.binding.enocean
+Bundle-ActivationPolicy: lazy
+Bundle-ClassPath: .
+Bundle-ManifestVersion: 2
+Bundle-Name: EnOcean Binding
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.binding.enocean;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.4.0.qualifier
+Export-Package: 
+ org.openhab.binding.enocean,
+ org.openhab.binding.enocean.handler,
+ org.openhab.binding.enocean.profiles
+Import-Package: 
+ com.google.common.collect,
```

Comment:
We try to get rid of Guava, so please try to do without this dependency.

Reply:
fixed